app/knights/knights.py

```diff
@@ -0,0 +1,19 @@
+class Knight:
+
+    def __init__(
+            self,
+            name: str,
+            power: int,
+            hp: int,
+            armour: list = None,
+            weapon: dict = None,
+            potion: int = None
+    ) -> None:
+
+        self.name = name
+        self.power = power
+        self.hp = hp
+        self.armour = armour
+        self.weapon = weapon
+        self.potion = potion
+        self.protection = 0

```

Comment:
so here you will use keys of this dict, i guess it's a better practice in this case😇